### PR TITLE
[Backport]fix for save on application close

### PR DESCRIPTION
**Pull request by elizabeth-legros**
Backport of https://github.com/Unity-Technologies/Graphics/pull/1252
https://fogbugz.unity3d.com/f/cases/1262231/
Tested closing unity with open dirty graph, could not repro

### CODE REVIEW

**Review by jessebarker — APPROVED** [review]
LGTM

File: com.unity.shadergraph/CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [7.6.0] - 2021-03-25
 
+### Fixed
```
Comment (marctem) [inline com.unity.shadergraph/CHANGELOG.md:9]: 7.6 already released - I'm not sure where to put this change, if we need a new 7.7 section or a 7.6.1. Probably need to ask in gfx-packages.

Comment (phi-lira) [reply com.unity.shadergraph/CHANGELOG.md:9]: I think that's not correct. Afaik we just release 7.5
If so this is the right place to add

File: com.unity.shadergraph/CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
+ - Fixed a bug where closing the editor with an open graph with changes would not notify the user to save the graph.
+
 Version Updated
 The version number for this package has increased due to a version update of a related graphics package.
```
Comment (phi-lira) [inline com.unity.shadergraph/CHANGELOG.md:13]: This should be removed.

File: com.unity.shadergraph/CHANGELOG.md (hunk 1 above)
Comment (sebastienlagarde) [reply com.unity.shadergraph/CHANGELOG.md:10]: we are 7.7 now as we will add support for Scarlet xbox. Master have been bumped, need to merge it, thanks

**Review by marctem — APPROVED** [review]

**Review by LandonTownsendUnity — APPROVED** [review]
Looks good!

**Pull request merged**